src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .split(';')
+    .filter((pair) => pair.trim() !== '');
```

Comment:
The chaining of `.split` and `.filter` is done correctly, but the `.filter` method can be optimized by using Boolean as a shorthand to remove empty strings after trimming, instead of explicitly comparing to an empty string.